iridauploader/model/metadata.py

```diff
@@ -0,0 +1,23 @@
+class OtherMetadata:
+
```

Comment:
Done, I believe it matches

Reply:
That schema is not a valid schema for `cerberus`, the model validator we use. I did some playing around with the validator to build a valid schema for this but it turns out since every entry is a wildcard, it doesn't really make much sense to use the schema in this way.

I think it would be best to just remove the schema after all.